### DISCUSSION TITLE
发送视频和语音

### DISCUSSION THREAD

**Discussion by MUYI-XIAN** [simple-robot/Discussions#73] [Q&A] [answered]
### 涉及的编程语言

Java

### 涉及的组件库

其他

### 疑问描述

请问一下视频和语言怎么发送？

**Comment by ForteScarlet** [comment, marked as answer]
mirai暂不支持视频发送 （[mirai#867](https://github.com/mamoe/mirai/issues/867)）
语音参考 https://github.com/simple-robot/simpler-robot/issues/400
